src/jit/codegenlinear.h

```diff
@@ -173,6 +173,14 @@ void genCodeForShiftLong(GenTreePtr tree);
 void genCodeForShiftRMW(GenTreeStoreInd* storeInd);
 #endif // _TARGET_XARCH_
 
+void genCodeForCast(GenTreeOp* tree);
+
+void genCodeForLclAddr(GenTree* tree);
+
+void genCodeForIndir(GenTreeIndir* tree);
+
+void genCodeForDiv(GenTreeOp* tree);
+
```

Comment:
It seems that there is a `genCodeForDivMod`defined in codegenlinear.h at https://github.com/dotnet/coreclr/blob/master/src/jit/codegenlinear.h#L23 but not used for arm and arm64.

How about merging `genCodeForDiv` with `genCodeForDivMod` ? or making one one of them is defined for armarch ?